pkg/kubelet/metrics/metrics.go

```diff
@@ -86,6 +86,10 @@ const (
 	// Metrics to track ephemeral container usage by this kubelet
 	ManagedEphemeralContainersKey = "managed_ephemeral_containers"
 
+	// Metrics to track the CPU manager behavior
+	CPUManagerPinningRequestsTotalKey = "cpu_manager_pinning_requests_total"
+	CPUManagerPinningErrorsTotalKey   = "cpu_manager_pinning_errors_total"
+
```

Comment:
What impact does adding these as kubelet metrics have if we ultimately decide to move the cpumanager outside of the kubelet as a pluggable component (as @catblade and others have been proposing).

Reply:
I can't say atm. it depends on the specifics of the move. Adding metrics to have observability of cpumanager was explicitely mentioned as GA requirement, so I believe that whatever direction we take to extract cpumanager & co, we will need to provide comparable metrics and observability features as prerequisite for GA.